### DISCUSSION TITLE
Spell checking on NixOS

### DISCUSSION THREAD

**Discussion by dzervas** [qutebrowser/qutebrowser#8284] [Q&A]
I'm trying to get spellchecking to work but I can't. I've set `c.spellcheck.languages = ["en-US", "el-GR"]` and I download the respective .bdic files to `${base}/data/qtwebengine_dictionaries/<name>.bdic`. qutebrowser doesn't nag for missing dictionaries but nothing happens - no underlying of incorrect words or suggestions. What am I missing?

I even tried to install hanspell but it did nothing

**Comment by The-Compiler** [comment]
What's `${base}/data` there? I'd recommend using the `scripts/dictcli.py` file coming with qutebrowser instead of doing things by hand.

**Reply by dzervas** [reply]
the problem with using dictcli is that I can't automate it in nixos so I'm downloading the bdics manually
`${base}` is the directory that is passed as `--basedir` in qutebrowser

**Comment by igsha** [comment]
I have NixOS. I used `dictcli.py` script from the qutebrowser installation location (`which qutebrowser | xargs readlink -f`). The script showed that it installed dictionaries into the `~/.local/share/qutebrowser/qtwebengine_dictionaries/` folder. Then, I set `spellcheck.languages` to the desired languages and restarted (!) qutebrowser. Now I can see incorrect words in edit fields.
